package.json

```diff
@@ -100,7 +102,7 @@
     "crowdin-clean": "rm -rf .crowdin && mkdir .crowdin",
     "crowdin-import": "ts-node src/scripts/crowdin-import.ts",
     "format": "prettier --write \"**/*.{js,jsx,json,md}\"",
-    "generate-heading-ids": "node src/scripts/generate-heading-ids.js",
+    "generate-heading-ids": "ts-node --esm src/scripts/generateHeadingIds.mts",
```

Comment:
@pettinarip 

I am using `esm` flag to mark it as ES Module along with `mts` extension for the file

Reply:
I had to manually set the syntax to `typescript` in neovim for this `mts` file to get the syntax highlighting.